src/js/core.js

```diff
@@ -149,7 +149,7 @@
             return;
         }
 
-        if (MediumEditor.util.isMediumEditorElement(node) && node.children.length === 0) {
+        if (MediumEditor.util.isMediumEditorElement(node) && node.children.length === 0 && !MediumEditor.util.isBlockContainer(node)) {
```

Comment:
Are you passing an element other than a `<div>` as the root editor element?

The call to `MediumEditor.util.isMediumEditorElement(node)` will only return true if `node` is the element that the editor is based on.  So, if you're using a `<div>` as the root element then this extra condition in the `if` won't actually do anything because it will always return `false` (since `<div>` is not in the list of `blockContainerElementNames`).